patch-tracing-channel.js

```diff
@@ -0,0 +1,163 @@
+const ReflectApply = Reflect.apply;
+const PromiseReject = Promise.reject;
+const PromiseResolve = Promise.resolve;
+const PromisePrototypeThen = Promise.prototype.then;
+const ArrayPrototypeSplice = Array.prototype.splice;
```

Comment:
This is incorrect as the first argument of the primordials in core for prototype functions is the instance it would be a method for, so the inverse of `bind(...)` essentially.

Something like this would be more correct:

```suggestion
const makeCall = (fn) => (...args) => fn.call(...args);
const PromisePrototypeThen = makeCall(Promise.prototype.then);
const ArrayPrototypeSplice = makeCall(Array.prototype.splice);
```